text/0172-health-api.md

```diff
@@ -0,0 +1,53 @@
+# Health API
+
+## 1. Summary
+
+This specification describes the health API endpoint. The `/health` route allows to verify the status and availability of a Meilisearch instance.
+
+## 2. Motivation
+
+To know the status of a Meilisearch instance, `/health` is a public route that can be long pooled with simple monitoring tools.
+
+## 3. Functional Specification
+
+### 3.1. `health` API resource properties
+
+| Field                                            | Type            | Required |
+|--------------------------------------------------|-----------------|----------|
+| [status](#311-status)                            | String          | True     |
+
+#### 3.1.1. `status`
+
+- Type: String
+- Required: True
+- Default: `available`
```

Comment:
```suggestion
- Value: `available`
```

I don't know what it's worth since you say it literally one line later, but we could be even more explicit by giving its value instead of its « default » value.

Reply:
Yes, this is a special case, there can be no other value than `available` so I put it in default to reuse an existing format in our specifications. It seemed appropriate but it can be changed!